tock/tock/static/js/components/timecard.js

```diff
@@ -363,6 +368,56 @@ function handleConfirm(val) {
   }
 }
 
+/** @function
+ * Show or hide the billing hour summation elements in the footer
+ * @name handleBillingElementState
+ * @param {string} visibility
+ * */
+function handleBillingElementState(visibility) {
+  const showBillingHourSummationElements = visibility === 'show';
+  const totalReportedElement = document.querySelector('#total-reported-div');
+  const totalBillableElement = document.querySelector('#total-billable-div');
+
+  if (showBillingHourSummationElements) {
+    const weeklyBillingAlertElement = document.querySelector('#weekly-billing-alert');
```

Comment:
**suggestion (non-blocking):** these two calls are repeated inside the conditional, but the ones for `totalReportedElement` and `totalBillableElement` are not. I think it's clearer to be consistent and pull all four of the `querySelector` calls outside the if statements and only make the changes in the if and else blocks.